py/server/tests/test_table.py

```diff
@@ -778,6 +780,66 @@ def test_python_field_access(self):
         self.assertIn("AAPL-GOOG", html_output)
         self.assertIn("2000", html_output)
 
+    def test_vectorization_exceptions(self):
```

Comment:
Seems like we should also be testing the numba vectorization here.

Reply:
We have separate test cases for numba in test_numba_xxx modules